plugins/photo_picker/README.md

```diff
@@ -0,0 +1,73 @@
+# Godot PhotoPicker plugin
+
+Example:
+
+```
+var _picker = null
+
+func _image_picked(image):
+	var texture = ImageTexture.new()
+	texture.create_from_image(image)
+	icon.texture = texture
+
+func _permission_updated(target, status):
+	match (target):
+		apns.PERMISSION_TARGET_PHOTO_LIBRARY:
+			print("photo library")
+		apns.PERMISSION_TARGET_CAMERA:
+			print("camera")
+	
+	match (status):
+		apns.PERMISSION_STATUS_UNKNOWN:
+			print("unknown")
+		apns.PERMISSION_STATUS_ALLOWED:
+			print("allowed")
+		apns.PERMISSION_STATUS_DENIED:
+			print("denied")
+            
+func _on_Button_button_down():
+	apns.present(apns.SOURCE_SAVED_PHOTOS_ALBUM);
+	apns.present(apns.SOURCE_CAMERA_REAR);
+	apns.present(apns.SOURCE_CAMERA_FRONT);
+    
+	print(apns.permission_status(apns.PERMISSION_TARGET_CAMERA))
+	print(apns.permission_status(apns.PERMISSION_TARGET_PHOTO_LIBRARY))
+    
+	apns.request_permission(apns.PERMISSION_TARGET_CAMERA)
+	apns.request_permission(apns.PERMISSION_TARGET_PHOTO_LIBRARY)
+
+...
+
+func _ready():
+	if Engine.has_singleton("PhotoPicker"):
+		_picker = Engine.get_singleton("PhotoPicker")
+		_picker.connect("image_picked", self, "_image_picked")
+		_picker.connect("permission_updated", self, "_permission_updated")
+		print("registering photo picker")
+	else:
+		print("no photo picker")
+```
+
+## Enum
+
+Type: `PhotoPickerSourceType`  
```

Comment:
There's some trailing whitespace on the "Type:" lines and in several lines of the code example which would be worth removing.